docs(releases): Add resolved in release documentation (#6168)
getsentry/sentry-docs: src/docs/product/releases/index.mdx

You can view release health data either from the <SandboxLink scenario="oneIssue" projectSlug="react">Issue Details</SandboxLink> page by selecting the commit ID listed under Last Seen", or from the **Releases** page.

> ## Resolve in release

Author: I'm not sure calling them release issues makes sense, makes it sound like a new product.

Different ways to use releases to resolve issues?

> ## Resolve in release

Author: thats what i ended up using!

Marking an issue as [resolved](/product/issues/states-triage/#resolve) in a release or in the next release allows you to mark an issue as resolved until the issue is seen again in a future release. Marking an issue as resolved in a release is different from just marking it resolved as resolved will unresolve on the next event, whereas resolved in release will only unresolve when the event is seen in a future release.

> When using "Resolved in the current release", Sentry records the current release, displayed in the ui, at the time of resolution. New events for this issue are compared with the recorded release version. If the release version is using semver, the issue will be marked as unresolved if the release is from a semver version that is greater than the resolved version. If the release is not using semver, the release creation dates are compared.

Reviewer: Upper-casing UI.
```suggestion
When using "Resolved in the current release", Sentry records the current release, displayed in the UI, at the time of resolution. New events for this issue are compared with the recorded release version. If the release version is using semver, the issue will be marked as unresolved if the release is from a semver version that is greater than the resolved version. If the release is not using semver, the release creation dates are compared.
```

You can view release health data either from the <SandboxLink scenario="oneIssue" projectSlug="react">Issue Details</SandboxLink> page by selecting the commit ID listed under Last Seen", or from the **Releases** page.

## Resolve in release

> 
> ![View of the resolve issue dropdown options on issue details.](resolved_in_options.png)
> 
> Marking an issue as [resolved](/product/issues/states-triage/#resolve) in a release or in the next release allows you to mark an issue as resolved until the issue is seen again in a future release.
> 
> If you'd like the issue to be marked as unresolved the next time an event is seen in any release, use the main "Resolve" button.
> If you'd like to ignore events until an event is seen in a future release, you can mark it resolved in the current release or the next release.
> 

Reviewer 2: ```suggestion
There are several options for resolving a release issue. If you'd like to leave the issue unresolved and be notified of it the next time any event is seen in a release, click the main "Resolve" button. But if you'd like to ignore the release issue until it's seen again in your current or next release, you can select "Resolved In The next release" or "Resolved In The current release" from the dropdown.

![View of the resolve issue dropdown options on issue details.](resolved_in_options.png)

.

```

> **Resolved in the current release** records the current release, displayed in the UI and new events for this issue are compared with the recorded release version to determine if the issue should unresolve. If the release version is using semver, the issue will be marked as unresolved if the release is from a semver version that is greater than the resolved version. If the release is not using semver, the release creation dates are compared.

Reviewer 2: ```suggestion
### Resolved in the Current Release

The "Resolved in the current release" option records the current release that's displayed in the UI. New events for the release issue are compared against the recorded release version to determine if the issue should unresolve. If the release version is using semver, the issue will be marked as unresolved if the release is from a semver version that's greater than the resolved version. If the release is not using semver, the release creation dates will be compared.
```

> **Resolved in the next release** works similar to resolved in current release except that we can't record the resolution version since it hasn't been created yet. Instead, we record the current release version and update it when a newer release is created.
> 

Reviewer 2: ```suggestion
### Resolved in the Next Release 

The "Resolved in the next release" option works in a similar way to resolved in the current release. But because we can't record the resolution version since it hasn't been created yet, we record the current release version and update it when a newer release is created.

```

> **Resolved in commit** waits for a release with the specified commit to be created. After the release is created, new events are checked against the release version or release date.
> 
> If the new event is from a newer release, the issue is marked as a regression and will appear in your [For Review tab](/product/issues/states-triage/) for triage.
> 

Reviewer 2: ```suggestion
### Resolved in a Commit

The "Resolved in a commit" option allows you to wait for a release with the specified commit to be created. Once the release is created, new events get checked against the release version or release date. If the new events are from a newer release, the issue gets marked as a regression and appears in your [For Review tab](/product/issues/states-triage/) to be triaged.

```
